packages/plugins/other/visitor-plugin-common/src/base-types-visitor.ts

```diff
@@ -456,7 +456,7 @@ export class BaseTypesVisitor<
   FieldDefinition(node: FieldDefinitionNode): string {
     const typeString = node.type as any as string;
     const { type } = this._parsedConfig.declarationKind;
-    const comment = this.getFieldComment(node);
+    const comment = this.getDeprecationComment(node);
```

Comment:
`getDeprecationComment(node)` is still handling some logic related to comments in general, not only deprecation.

Please leave it as `getNodeComment()` 🙂 

```suggestion
    const comment = this.getNodeComment(node);
```